game-core/src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -484,14 +485,12 @@ private void setWhenCapturedChangesInto(final String value) throws GameParseExce
     }
     getBool(s[2]);
     final IntegerMap<UnitType> unitsToMake = new IntegerMap<>();
-    for (int i = 3; i < s.length; i++) {
+    for (int i = 3; i < s.length; i += 2) {
       final UnitType ut = getData().getUnitTypeList().getUnitType(s[i]);
       if (ut == null) {
-        throw new GameParseException("whenCapturedChangesInto: No unit named: " + s[3] + thisErrorMsg());
+        throw new GameParseException("whenCapturedChangesInto: No unit named: " + s[i] + thisErrorMsg());
```

Comment:
This is the only functional change.
In the case of multiple units the error message would make no sense at all.

Reply:
Yeah, pretty sure that is a mistake.